data/steam.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "steam",
+  "name": "Steam",
+  "description": "Digital distribution platform by Valve ",
+  "categories": [],
+  "iconUrl": "https://i.imgur.com/IskAj8F.jpg",
+  "website": "https://store.steampowered.com",
+  "applicationUrl": "https://store.steampowered.com/",
+  "countryHQ": "US",
+  "gdprReadyStatus": "ready",
```

Comment:
Did you read an official of this? If yes, please add this to the articles. If no I would choose inProgress.

Reply:
It's in their privacy policy ............... https://store.steampowered.com/privacy_agreement/ :/